spec/models/pawn_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe "Pawn#valid_move?(destination_x, destination_y)" do
+    
+    let(:white_player) { FactoryGirl.create(:player, id: 100, email: 'blah@blah.com', password: 'SPACECAT') }
+    let(:pawn) { FactoryGirl.create(:pawn, game_id: game.id, player_id: white_player.id) }
+    let(:game) { FactoryGirl.create(:game) }
+    
+    subject { pawn.valid_move?(destination_x, destination_y) }
+    
+    context "valid move for first move" do
+      let(:destination_x) { 1 }
+      let(:destination_y) { 2 }
+      
+      it "should return true if the white pawn is moving 1 or 2 squares up for first move" do
+        expect(subject).to eq(true)
+      end
+    end
```

Comment:
Sure, I can do that. Right now, I have a test for the first move - that includes 1 or 2 space moves since that is what a pawn can do. I can do a test specifically for two moves to start.